src/screens/history/index.tsx

```diff
@@ -1,11 +1,39 @@
-import React from 'react';
-import {Text} from 'react-native';
+import MatchCard from 'components/shared/matchCard';
+import React, {useEffect} from 'react';
+import {FlatList, Text, View} from 'react-native';
 import {SafeAreaView} from 'react-native-safe-area-context';
+import {useDispatch, useSelector} from 'react-redux';
+import {getMatches} from 'store/modules/match/actions';
+import {Match} from 'store/modules/match/reducers';
+import {matchData} from 'store/modules/match/selectors';
+import {MatchStatus} from 'types/enums/match';
+import styles from './styles';
 
 const History = () => {
+  const {matches} = useSelector(matchData);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMatches());
+  }, [dispatch]);
+
+  const completedMatches = matches.filter(match => {
+    return match.status !== MatchStatus.toBePlayed;
+  });
```

Comment:
We can add this logic to the selector, and named it something like `historyMatchesData`, what do you think?